### PR TITLE
Welding Fuel is now labelled Dr.Gibb when in a glass

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Apparently the sprites were the same so fuck it this'll be funny every now and then

# Changelog

:cl:  
tweak: Welding Fuel is now labelled as Dr.Gibb when in a glass
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1057,8 +1057,8 @@
 	color = "#660000" // rgb: 102, 0, 0
 	taste_description = "gross metal"
 	glass_icon_state = "dr_gibb_glass"
-	glass_name = "glass of welder fuel"
-	glass_desc = "Unless you're an industrial tool, this is probably not safe for consumption."
+	glass_name = "glass of Dr. Gibb"
+	glass_desc = "Dr. Gibb. Not as dangerous as the glass_name might imply."
```
Comment (Moltijoe) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:1061]: glass_name?

**Comment by Moltijoe** [comment]
RP buff to robotic species

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (ynot01) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1061]: wat?

Comment (Moltijoe) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1061]: "Not as dangerous as the glass_name might imply"

Comment (ynot01) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1061]: Some meta reference I suppose? This is what's written on Dr.Gibb

Comment (Moltijoe) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1061]: hmm, fair enough

**Review by Moltijoe — APPROVED** [review]

**Comment by iamthesilly** [comment]
oh my god i love this

**Pull request merged**